StandardConfig/production/MarlinStdRecoMiniDST.xml

```diff
@@ -817,14 +824,10 @@
   <parameter name="FlavorTag.JetCollectionName" type="string" value="Refined6Jets" />
   <parameter name="MakeNtuple.AuxiliaryInfo" type="int" value="-1" />
 
-  <!--parameter name="FlavorTag.WeightsDirectory" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/lcfiweights"/--> <!-- e.g. lcfiweights/lcfiweights_6q500_v04_p00_ildl5 -->
-  <!--parameter name="FlavorTag.WeightsPrefix" type="string" value="6q500_v04_p00_ildl5" /--> <!-- e.g. 6q500_v04_p00_ildl5 -->
-  <parameter name="FlavorTag.WeightsDirectory" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/v020201/lcfiweights/4q250_ZZ_v4_p00_ildl5"/>
-  <parameter name="FlavorTag.WeightsPrefix" type="string" value="4q250_v04_p00_ildl5" />
+  <parameter name="FlavorTag.WeightsDirectory" type="string" value="${LCFIPlusWeightsDir}"/>
+  <parameter name="FlavorTag.WeightsPrefix" type="string" value="${LCFIPlusWeightsPrefix}"/>
   <parameter name="FlavorTag.BookName" type="string" value="bdt" />
   <parameter name="FlavorTag.PIDAlgo" type="string" value="lcfiplus" />
-  <!--parameter name="FlavorTag.D0ProbFileName" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/vtxprob/d0probv2_ildl5_6q500.root"/--> <!-- vtxprob/d0probv2_ildl5_6q500.root -->
-  <!--parameter name="FlavorTag.Z0ProbFileName" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/vtxprob/z0probv2_ildl5_6q500.root"/--> <!-- vtxprob/z0probv2_ildl5_6q500.root -->
   <parameter name="FlavorTag.D0ProbFileName" type="string" value="/cvmfs/ilc.desy.de/sw/ILDConfig/v02-02-01/LCFIPlusConfig/vtxprob/d0probv2_ildl5_4q250_ZZ.root"/>
```

Comment:
Could we make these also constants ? 
Could set the path to $PWD and add the corresponding copy and unpack command and how to run it to [https://github.com/iLCSoft/ILDConfig/tree/master/StandardConfig/production#readme](https://github.com/iLCSoft/ILDConfig/tree/master/StandardConfig/production#readme).
This then also serves as documentation and introduction on how to run everything ....

Reply:
... all necessary files are in `$ILDCONFIG_PATH/LCFIPlusConfig`, i.e. in `../../LCFIPlusConfig` from the current production dir...

Reply:
I have introduced a few more `constants` and tried to give them meaningful defaults, such that minimal example can be run with specifying `ILDConfig_DIR` (assuming unpacked weights)